### ISSUE TITLE
[BUG] MissingPluginException when showInspectorOnShake:true 

### ISSUE THREAD

**Issue by fairuzmn** [jhomlala/alice#166]
**Describe the bug**

**To Reproduce**

When showInspectorOnShake true

```
final Alice alice = Alice(showNotification: true, showInspectorOnShake: true, notificationIcon: "@mipmap/ic_launcher");
```

Got error with log

```
E/flutter (17356): [ERROR:flutter/runtime/dart_vm_initializer.cc(41)] Unhandled Exception: MissingPluginException(No implementation found for method setAccelerationSamplingPeriod on channel dev.fluttercommunity.plus/sensors/method)
E/flutter (17356): #0      MethodChannel._invokeMethod (package:flutter/src/services/platform_channel.dart:320:7)
E/flutter (17356): <asynchronous suspension>
E/flutter (17356): 
```

**Expected behavior**

**Screenshots**

**Flutter doctor**
```
# flutter doctor -v
[√] Flutter (Channel stable, 3.16.7, on Microsoft Windows [Version 10.0.22631.3007], locale en-ID)
    • Flutter version 3.16.7 on channel stable at D:\utils\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision ef1af02aea (4 days ago), 2024-01-11 15:19:26 -0600
    • Engine revision 4a585b7929
    • Dart version 3.2.4
    • DevTools version 2.28.5

[√] Windows Version (Installed version of Windows is version 10 or higher)

[√] Android toolchain - develop for Android devices (Android SDK version 34.0.0)
    • Android SDK at C:\Users\Parasit\AppData\Local\Android\sdk
    • Platform android-34, build-tools 34.0.0
    • Java binary at: C:\Program Files\Android\Android Studio\jbr\bin\java
    • Java version OpenJDK Runtime Environment (build 17.0.7+0-b2043.56-10550314)
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[!] Visual Studio - develop Windows apps (Visual Studio Build Tools 2019 16.11.29)
    • Visual Studio at C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools
    • Visual Studio Build Tools 2019 version 16.11.33927.289
    X The current Visual Studio installation is incomplete.
      Please use Visual Studio Installer to complete the installation or reinstall Visual Studio.

[√] Android Studio (version 2023.1)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 17.0.7+0-b2043.56-10550314)

[√] VS Code, 64-bit edition (version 1.84.2)
    • VS Code at C:\Program Files\Microsoft VS Code
    • Flutter extension version 3.80.0

[√] Connected device (4 available)
    • M2010J19CG (mobile) • 5f5300d91120 • android-arm64  • Android 14 (API 34)
    • Windows (desktop)   • windows      • windows-x64    • Microsoft Windows [Version 10.0.22631.3007]
    • Chrome (web)        • chrome       • web-javascript • Google Chrome 120.0.6099.217
    • Edge (web)          • edge         • web-javascript • Microsoft Edge 120.0.2210.121

[√] Network resources
    • All expected network resources are available.

! Doctor found issues in 1 category.

```

**Alice version**
- Version: ^0.4.1

**Smartphone (please complete the following information):**
 - Device: Poco M3 
 - OS: Android 13 - 14

**Additional context**

Even it show error, the shake function still working, but the error log is still there until i set showInspectorOnShake to false.
Any idea?

**Comment by danuprakoso26** [comment]
same, did you find a solution? @fairuzmn

**Comment by fairuzmn** [comment]
not yet

**Comment by danuprakoso26** [comment]
try add sensors_plus: 2.0.1 on pubspecs

**Comment by andyshephard** [comment]
It's happening because the `sensors_plus` package being used is old and needs to be updated to the next major version. Other packages are experiencing similar issues (can see here: https://github.com/fluttercommunity/plus_plugins/issues/2471)

There's already a PR waiting to be reviewed & merged which will fix this: https://github.com/jhomlala/alice/pull/159

**Comment by husainazkas** [comment]
> It's happening because the `sensors_plus` package being used is old and needs to be updated to the next major version. Other packages are experiencing similar issues (can see here: [fluttercommunity/plus_plugins#2471](https://github.com/fluttercommunity/plus_plugins/issues/2471))
> 
> There's already a PR waiting to be reviewed & merged which will fix this: #159

If it takes a lot more time to waiting for merge PR, you can override this dependency using `dependency_overrides`

I think this issue can be closed.

**Comment by blingkonnerth** [comment]
> It's happening because the `sensors_plus` package being used is old and needs to be updated to the next major version. Other packages are experiencing similar issues (can see here: [fluttercommunity/plus_plugins#2471](https://github.com/fluttercommunity/plus_plugins/issues/2471))
> 
> There's already a PR waiting to be reviewed & merged which will fix this: #159

Still experiencing this issue. Any news?

**Comment by jhomlala** [comment]
It should be fixed with latest release.
